clang/test/dpct/cmake_migration/case_063/input.cmake

```diff
@@ -0,0 +1,9 @@
+find_program(CUDAToolkit_NVCC_EXECUTABLE nvcc ${CUDAToolkit_BIN_DIR})
+
+find_program(CUDAToolkit_NVCC_EXECUTABLE
+    NAMES nvcc nvcc.exe
```

Comment:
what happen if nvcc_other, will it be replace by $(CMAKE_CXX_COMPILER)_other ?

Reply:
Yes , I think for nvcc_exe can be mapped to $(CMAKE_CXX_COMPILER) ?

Reply:
Lib names of form **nvcc_other** will not be migrated to "${CMAKE_CXX_COMPILER}_other"

As the **MatchMode** is **Full**, the Input string is matched for the entire CMake arg until a valid separator is encountered

Reply:
if nvcc.other can be migrated to ${...}.other,  then  it is a bug, (partial match).

Reply:
Please re-review once.